applications/nrf_desktop/README.rst

```diff
@@ -1512,6 +1512,14 @@ For example, the following line will start the upload of the new image to the de
 
 ----
 
+.. options-from-kconfig::
```

Comment:
@greg-fer Could we keep it in a separate file?

Reply:
I would prefer to stick to keeping them on one page, but at the very bottom? Would this make sense?

Reply:
Sure, will move the to the bottom of the file and add heading

Reply:
done